spring-cloud-stream-modules-docs/src/main/asciidoc/sinks.adoc

```diff
@@ -1,6 +1,58 @@
 [[spring-cloud-stream-modules-sinks]]
 == Sinks
 
+[[spring-cloud-stream-modules-cassandra]]
+=== Cassandra (`cassandra`)
+
+The Cassandra sink writes into a Cassandra table.  Here is a simple example
+
+  dataflow:>stream create cassandrastream --definition "http --server.port=8888 --spring.cloud.stream.bindings.output.contentType='application/json' | cassandra --ingestQuery='insert into book (id, isbn, title, author) values (uuid(), ?, ?, ?)' --spring.cassandra.keyspace=clouddata" --deploy
+
+Create a keyspace and a `book` table in Cassandra using:
+
+[source,text]
+----
+CREATE KEYSPACE clouddata WITH REPLICATION = { 'class' : 'org.apache.cassandra.locator.SimpleStrategy', 'replication_factor': '1' } AND DURABLE_WRITES = true;
+USE clouddata;
+CREATE TABLE book  (
+    id          uuid PRIMARY KEY,
+    isbn        text,
+    author      text,
+    title       text
+);
+----
+
+You can then send data to this stream via
+
+  dataflow:>http post --contentType 'application/json' --data '{"isbn": "1599869772", "title": "The Art of War", "author": "Sun Tzu"}' --target http://localhost:8888/messages
+  > POST (application/json;charset=UTF-8) http://localhost:8888/messages {"isbn": "1599869772", "title": "The Art of War", "author": "Sun Tzu"}
+  > 202 ACCEPTED
+
+and see the table contents using the CQL
+
+  SELECT * FROM clouddata.book;
+
+==== Options
+
+The **$$cassandra$$** $$sink$$ has the following options:
+
+$$compressionType$$:: $$the compression to use for the transport$$ *($$CompressionType$$, default: `NONE`, possible values: `NONE,SNAPPY`)*
+$$consistencyLevel$$:: $$the consistencyLevel option of WriteOptions$$ *($$ConsistencyLevel$$, no default, possible values: `ANY,ONE,TWO,THREE,QUOROM,LOCAL_QUOROM,EACH_QUOROM,ALL,LOCAL_ONE,SERIAL,LOCAL_SERIAL`)*
+$$spring.cassandra.contactPoints$$:: $$the comma-delimited string of the hosts to connect to Cassandra$$ *($$String$$, default: `localhost`)*
+$$entityBasePackages$$:: $$the base packages to scan for entities annotated with Table annotations$$ *($$String[]$$, default: `[]`)*
+$$spring.cassandra.ingestQuery$$:: $$the ingest Cassandra query$$ *($$String$$, no default)*
```

Comment:
The `ingestQuery` is a part of `Sink`, so it is without prefix.
See https://github.com/spring-cloud/spring-cloud-stream-modules/blob/master/cassandra-sink/src/main/java/org/springframework/cloud/stream/module/cassandra/CassandraProperties.java for all with prefix.